### ISSUE TITLE
Multiple button presses to send email not recognized

### ISSUE THREAD

**Issue by ecki** [softwerkskammer/Agora#1410]
I just sent a mail via Agora and since it did not immediately recognized the send button press (at least it seemed that way on the Tablet) I pressed the button a few times and it did send me 3 duplicates of the message, 

Client JS should probably disable the button and give feedback, and the server should accept the form only once.
